queries/d/highlights.scm

```diff
@@ -1,22 +1,79 @@
 ;; Misc
 
-[
-  (line_comment)
-  (block_comment)
-  (nesting_block_comment)
-] @comment @spell
+(comment) @comment @spell
 
-((line_comment) @comment.documentation
+((comment) @comment.documentation
   (#lua-match? @comment.documentation "^///[^/]"))
-((line_comment) @comment.documentation
+
+((comment) @comment.documentation
   (#lua-match? @comment.documentation "^///$"))
 
-((block_comment) @comment.documentation
+((comment) @comment.documentation
   (#lua-match? @comment.documentation "^/[*][*][^*].*[*]/$"))
 
-((nesting_block_comment) @comment.documentation
+((comment) @comment.documentation
   (#lua-match? @comment.documentation "^/[+][+][^+].*[+]/$"))
 
+(directive) @preproc
+(shebang) @preproc
+
+;; Operators
+
+; these are listed first, because they override keyword queries
+(identity_expression (in) @operator)
```

Comment:
`is`/`in` are they literal `in`/`is` in the language? Then perhaps `@keyword.operator`? What about `not_in`

Reply:
Done